src/components/personkort/PersonkortLege.tsx

```diff
@@ -37,14 +37,26 @@ export const hentTekstFastlegePostadresse = (postadresse: Adresse) => {
     : "";
 };
 
+const FastlegeVikarList = styled.ul`
+  span {
+    margin-right: 1.5em;
+  }
+`;
+
 const fastlegeVikarTekst = (fastlegeVikar: Fastlege) => {
   const vikarlegeNavn = hentTekstFastlegeNavn(fastlegeVikar);
   const periodeTekst = `${restdatoTilLesbarDato(
     fastlegeVikar.gyldighet.fom
   )} - ${restdatoTilLesbarDato(fastlegeVikar.gyldighet.tom)}`;
+  const stillingsprosentTekst =
+    fastlegeVikar.stillingsprosent && `${fastlegeVikar.stillingsprosent}%`;
   return (
     <>
-      <b>{vikarlegeNavn}</b> {periodeTekst}
+      <span>
+        <b>{vikarlegeNavn}</b>
+      </span>
+      <span>{periodeTekst}</span>
+      {stillingsprosentTekst && <span>{stillingsprosentTekst}</span>}
```

Comment:
Ble noen span her nå, men det er kanskje noe annet som er bedre rent semantisk? Liste i liste? Tar gjerne imot innspill her.

Reply:
Kunne kanskje brukt FlexRow / FlexColumn her for å slippe span og gjøre elementene mer "alignet"

Reply:
Jeg prøvde å gjøre det litt likt en skisse Espen lagde tidligere, men jeg prøver Flex og ser hvordan det blir +1

Reply:
Byttet til FlexRow/FlexColumn, men ellers lik styling så det er som den skissen:
https://www.figma.com/file/L6DPvt6dU1XzypeiQoQ54C/MODIA-Person?node-id=6187:37568